src/types/breakpoints.ts

```diff
@@ -10,16 +9,14 @@ type ExtractTransformArray<T> = T extends object
 type ExtractBreakpoints<T> = T extends object
     ? keyof T extends BreakpointsOrMediaQueries
         ? T[keyof T]
-        : T extends Array<ToDeepUnistyles<TransformStyles>>
-            ? Array<ExtractTransformArray<T[number]>>
-            : {
-                [K in keyof T]: ExtractBreakpoints<T[K]>
-            }
-    : T
+        : T
+    : T extends Array<ToDeepUnistyles<TransformStyles>>
+        ? Array<ExtractTransformArray<T[number]>>
+        : T
```

Comment:
This needs to be updated to handle the other nested styles ('shadowOffset' | 'transform' | 'textShadowOffset' | 'boxShadow' | 'filter')